docs/userguide/index.de.md

```
@@ -8,13 +8,14 @@ Herzlich willkommen! Dieses Handbuch führt dich durch alles, was dieses BBS zu
 2. [Echomail: Globale Foren](#echomail-globale-foren)
-3. [Netmail: Private Nachrichten](#netmail-private-nachrichten)
-4. [Lokaler und vernetzter Chat](#lokaler-und-vernetzter-chat)
-5. [Doors: Spiele & Anwendungen](#doors-spiele-anwendungen)
-6. [Dateibereiche](#dateibereiche)
-7. [Die BBS-Netzwerke erkunden](#die-bbs-netzwerke-erkunden)
-8. [Dein Profil & Einstellungen](#dein-profil-einstellungen)
-9. [Hilfe erhalten](#hilfe-erhalten)
+3. [Nachrichtenleser](#nachrichtenleser)
+4. [Netmail: Private Nachrichten](#netmail-private-nachrichten)
+5. [Lokaler und vernetzter Chat](#lokaler-und-vernetzter-chat)
+6. [Doors: Spiele & Anwendungen](#doors-spiele-anwendungen)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)